README.md

```diff
@@ -93,6 +93,10 @@ It is based upon and contains a lot of code written by [Jeroen De Dauw]
 
 ## Release notes
 
+### 1.1.7 (2016-05-25)
+
+* Fixed `FORMAT_NAME` string constants in all parsers
```

Comment:
Refactoring? Anyway, regardless of the precise classification, I'd not mention it in the release notes, as I don't see how it is of interest to users of this component. And if it is, then one needs to provide the info they need.